docker-run.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-INDEX_FILE=/usr/share/nginx/html/index.html
+INDEX_FILE=/usr/share/nginx/html/${BASE_PATH}index.html
```

Comment:
to be clear, this won't affect the _public_ hosting location, right?  This organization is only for _internal_ purposes?